micronaut-graalpy-bom/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id 'io.micronaut.build.internal.graal-languages-base'
+    id "io.micronaut.build.internal.bom"
+}
+
+micronautBuild {
```

Comment:
don't remember how i came up with this, but without it build failed with 
```
Execution failed for task ':micronaut-graalpy:findBaseline'.
> Could not find a previous version for 1.0.0
```
alternatively, 
`findBaseline.enabled = false`
also prevented the error